infra/prod/italynorth/locals.tf

```
@@ -0,0 +1,19 @@
+locals {
+  prefix               = "io"
+  env_short            = "p"
+  location_short       = "itn"
+  location             = "italynorth"
+  project              = "${local.prefix}-${local.env_short}-${local.location_short}"
+  is_prod              = local.env_short == "p" ? true : false
+  application_basename = "services-cms"
```

suggestion (blocking): keep `application_basename` more generic

@giuseppedipinto @giamma1295 @StefanoDoc do you agree to keep `application_basename` more generic and add, for example, `-cms` or `-app-be` to specific resources?
I mean, the resources in this PR are not related to Services CMS, so I think we should not add `-cms` suffix to them to keep their names clean.
Maybe, we can add two more vars (i.e. `application_basename_cms` and `application_basename_app`) composed by `application_basename.concat("<specific_suffix>")`
```suggestion
  application_basename = "services"
```

---

I agree, services-cms is too specific for a single application in this monorepo, services or io-services would be better, however the repo name is "io-services-cms", so we should rename it to tend as well, to somenthing more generic like "io-services"